saleor/dashboard/discount/forms.py

```diff
@@ -21,6 +21,22 @@ class SaleForm(forms.ModelForm):
     class Meta:
         model = Sale
         exclude = []
+        labels = {
+            'name': pgettext_lazy(
+                'Sale form label',
+                'Sale name'),
+            'type': pgettext_lazy(
+                'Sale form label',
+                'Type of products related to the sale'),
```

Comment:
Isn't it rather discount type, fixed or percentage?

Reply:
Right, fixed.